src/QuizAppMainApi/Repositories/Quests/QuestRepository.cs

```diff
@@ -44,5 +44,14 @@ public async Task AddQuest(QuestInformation questInformation)
 
             await _context.SaveChangesAsync();
         }
+
+        public async Task<IEnumerable<QuestInformation>> GetTenQuests()
+        {
+            var Quests = await _context.Quests.
```

Comment:
Don't take 10 random quizzes from each category! You should pass categoryId and select 10 quizzes with same categoryId